.gitmodules

```diff
@@ -145,6 +145,10 @@
 [submodule "third_party/mimalloc"]
 	path = third_party/mimalloc
 	url = https://github.com/microsoft/mimalloc.git
+[submodule "third_party/composable_kernel"]
+	path = third_party/composable_kernel
+	url = https://github.com/ROCm/composable_kernel.git
+	branch = develop
```

Comment:
Why the `branch = develop` is this pointing to an unstable pin?

Reply:
The universal gemm CK templates I'm using are relatively new (end of April) and we do not have a stable release which contains them yet. If I understand correctly the git submodules, we can do independent update (git pull) of the composable kernel submodule and commit it into the pytorch repository (e.g. see https://github.com/tenpercent/pytorch/commit/5a2763fb2f9489048a78085b65a73067661f35be)